wordpress-mobile/WordPress-Android
# [Gutenberg ]Circle Progress for MediaUploadProgress [Testing PR]

## Testing
1. Create a new post. 
2. Add a File Block. 
3. Choose a file from the device. 
4. Notice the circle progress circle updates. 

## Screenshots 

<kbd><img src="https://user-images.githubusercontent.com/1509205/100815018-07824b80-3411-11eb-977c-6071b1b7d24f.png" width="320"></kbd>


## Noticeable issues

1. The progress component flickers when it's first drawn. 
2. The RichText component wraps on Android. Similar issue to the one that occurred with the placeholder issue.
3. Margin needs to be added to the component. 
4. If you type a really long title the progress circle goes off-screen so we would need to have some form of flexbox styling that stops the `RichText` component from growing to the edge of the screen. 

PR submission checklist:
